Source/OpenDRIVE/Private/OpenDRIVEActor.cpp

```diff
@@ -15,14 +15,16 @@ AOpenDRIVEActor::AOpenDRIVEActor()
 {
 	// Set this actor to call Tick() every frame.  You can turn this off to improve performance if you don't need it.
 	PrimaryActorTick.bCanEverTick = false;
-
+	SetRootComponent(CreateDefaultSubobject<USceneComponent>(TEXT("DefaultRootComponent")));
 }
 
 // Called when the game starts or when spawned
 void AOpenDRIVEActor::BeginPlay()
 {
 	Super::BeginPlay();
-	LoadOpenDrive();
+	
+	CheckForMultipleOpenDRIVEActors() ? GEngine->AddOnScreenDebugMessage(0, 10.f, FColor::Red, TEXT("Multiple OpenDRIVEActor detected. Must be only OpenDRIVEActor per scene. \n The plugin can't work correctly."))
```

Comment:
Also, maybe look at `UViewportStatsSubsystem`, which is supposed to allow us to add more permanent messages on the viewport. Because `AddOnScreenDebugMessage` has a specific duration which doesn't really fit our requirement: we want the message to stay on until the issue is fixed.